### PR TITLE
Bookstore: actions and reducers

### PR DESCRIPTION
**Pull request by AHP15**
In this pull request I added the following:

 - Installed Redux Toolkit `npm install react-redux @reduxjs/toolkit`
 - Created folder structure following the `duck` pattern
 - Added the book's actions and reducer.
 - Added the categories' actions and reducer.
 
Cheers :)

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// action types
+const LOAD_BOOKS = 'load';
+const ADD_BOOK = 'add-book';
+const REMOVE_BOOK = 'remove-book';
```
Comment (omarsalem7) [inline src/redux/books/books.js]: - [ ] Kindly consider the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). For example, `const ADD_BOOK = 'bookstore/books/ADD_BOOK'`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// action types
+const STATUS = 'status';
```
Comment (omarsalem7) [inline src/redux/categories/categories.js]: - [ ] As stated before, Kindly consider the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Abdessittir 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️

- [ ] Kindly fix the license MIT link in the README and add your own license.
- [ ] _Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AHP15** [comment]
Hi @omarsalem7 thank you for the feedback, I will make those changes

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Abdessittir 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

> And Congrats :confetti_ball: :champagne: :balloon: to Morocco ⚽

TO HIGHLIGHT:

- Great job implementing the changes provided by the previous reviewer ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AHP15** [comment]
Hi @Bondok6, thank you for the two congratulations ❤️

**Pull request merged**